src/Microsoft.DocAsCode.MarkdigEngine.Extensions/Image/Image.cs

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Microsoft. All rights reserved.
```

Comment:
Since you add a new extension contain path need to resolve, you may also need to change this file https://github.com/dotnet/docfx/blob/dev/src/Microsoft.DocAsCode.MarkdigEngine/ResolveLink/ResolveLinkExtension.cs#L52 . Or when the extension is written in a token, the path will not be resolved correctly.

Reply:
In the same way, you may also need to add a test case about new syntax in token, to check whether the image path is resolved successfully.

Reply:
Can you clarify? I'm not sure I understand what change needs to be done to https://github.com/dotnet/docfx/blob/dev/src/Microsoft.DocAsCode.MarkdigEngine/ResolveLink/ResolveLinkExtension.cs#L52

Reply:
Sure. You can take a look about this PR https://github.com/bharney0/docfx/pull/1/files. It include one unit test and the related code changes. The code should work but you may need to do some refactor~